pkg/api/api_test.go

```diff
@@ -180,11 +256,191 @@ var _ = Describe("Api", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				rr := httptest.NewRecorder()
-				handler := http.HandlerFunc(catalogAPI.ProfileHandler)
+				handler := http.HandlerFunc(catalogAPI.ProfileWithVersionHandler)
```

Comment:
this test was completely broken...

Reply:
How was it not caught? Why wasn't it failing? :O